newEvaluation.html

```diff
@@ -379,10 +380,14 @@
 					</li>
 				</ul>
 			</fieldset>
-			<button type = "submit" name = "submit">Submit</button>
+			<button id = "submitButton" type = "submit" name = "submit">SUBMIT</button>
```

Comment:
What if I told you that you could affect the casing of a piece of text through CSS alone? What if I also told you that you could adjust the spacing around elements with CSS, as well, and that tags like `<br>` are a bad practice, since they force information at the data layer to be presented a certain way across all environments / devices / screen sizes etc? What if I also told you that for mobile devices with less screen real estate, the presentation often differs slightly, so as to be able to fit more information in less space?